package.json

```diff
@@ -726,7 +726,7 @@
               },
               "buildTarget": {
                 "type": "string",
-                "description": "Name of the build target of the class to run"
+                "description": "Name of the build target of the class to run, can be found in the projects section of the Metals tab"
```

Comment:
I added a bit more info here, which gets displayed when you mouse over the build target. I feel like including the description in the value might be misleading.

Reply:
Good idea!